diff -Naur sympy-0.7.5.orig/setup.py sympy-0.7.5/setup.py
--- sympy-0.7.5.orig/setup.py	2014-05-15 15:24:32.724322914 +0400
+++ sympy-0.7.5/setup.py	2014-05-15 15:24:46.454322226 +0400
@@ -70,11 +70,6 @@
     'sympy.logic.utilities',
     'sympy.matrices',
     'sympy.matrices.expressions',
-    'sympy.mpmath',
-    'sympy.mpmath.calculus',
-    'sympy.mpmath.functions',
-    'sympy.mpmath.libmp',
-    'sympy.mpmath.matrices',
     'sympy.ntheory',
     'sympy.parsing',
     'sympy.physics',
@@ -240,7 +235,6 @@
     'sympy.logic.tests',
     'sympy.matrices.expressions.tests',
     'sympy.matrices.tests',
-    'sympy.mpmath.tests',
     'sympy.ntheory.tests',
     'sympy.parsing.tests',
     'sympy.physics.hep.tests',
